bin/ch/HostConfigFlags.cpp

```diff
@@ -96,6 +96,7 @@ void HostConfigFlags::RemoveArg(int& argc, _Inout_updates_to_(argc, argc) LPWSTR
     Assert(index >= 0 && index < argc);
     for (int i = index + 1; i < argc; ++i)
     {
+#pragma prefast(suppress:6385, "Operation is safe but PREfast is difficult to convince")
```

Comment:
>6385 [](start = 25, length = 4)

suppress warning by name macro instead of number?

Reply:
Done in next set of commits.